pages/checkout.js

```diff
@@ -0,0 +1,123 @@
+const { I } = inject();
+
+module.exports = {
+checkoutHeaderText: 'Checkout',
+newBillingAddressButton: {xpath: '//*[@id="collapse-payment-address"]/div/form/div[3]/label'},
+firstNameBillingField: {css: '#input-payment-firstname'},
+lastNameBillingField: {css: '#input-payment-lastname'},
+companyBillingField: {css: '#input-payment-company'},
+address1BillingField: {css: '#input-payment-address-1'},
+address2BillingField: {css: '#input-payment-address-2'},
+cityBillingField: {css: '#input-payment-city'},
+postcodeBillingField: {css: '#input-payment-postcode'},
+regionStateButton: {xpath: '//a[contains(text(), "--- Please Select ---")]'},
+regionSelectorChoice: {xpath: '//a[contains(text(), "Aberdeen")]'},
+continueOnCheckoutPageButton: {xpath: '//*[@id="button-payment-address"]'},
+continueForDeliveryDetails: {xpath: '//*[@id="button-shipping-address"]'},
+continueDeliveryMethodButton: {xpath: '//*[@id="button-shipping-method"]'},
+termsAndConditionsAgreementButton: {xpath: '//*[@id="agree1"]'},
+continueForPaymentMethod: {xpath: '//*[@id="button-payment-method"]'},
+flatShippingRatePriceText: {xpath: '//*[@id="collapse-checkout-confirm"]/div/div[1]/table/tfoot/tr[2]/td[2]'},
+ecoTaxPriceText: {xpath: '//*[@id="collapse-checkout-confirm"]/div/div[1]/table/tfoot/tr[3]/td[2]'},
+vatPriceText: {xpath: '//*[@id="collapse-checkout-confirm"]/div/div[1]/table/tfoot/tr[4]/td[2]'},
+totalPriceText: {xpath: '//*[@id="collapse-checkout-confirm"]/div/div[1]/table/tfoot/tr[5]/td[2]'},
+confirmOrderButton: {xpath: '//*[@id="button-confirm"]'},
+successfulOrderText: 'Your order has been placed!',
+orderIdText: {xpath: '//*[@id="content"]/div[1]/table/tbody/tr[1]/td[1]'},
+myAccountButton: {xpath: '//*[@id="top-links"]/ul/li/span/span'},
+orderHistoryButton: {xpath: '//*[@id="top-links"]/ul/li/ul/li[2]/a'},
```

Comment:
use text as starter point, that will be more unique

Reply:
fixed